plugins/main/public/components/overview/vulnerabilities/events/vulnerabilities-columns.tsx

```diff
@@ -19,4 +21,7 @@ export const vulnerabilitiesColumns: tDataGridColumn[] = [
   {
     id: 'data.vulnerability.status',
   },
+  {
+    id: 'data.vulnerability.package.version',
+  },
```

Comment:
**documentation**: This change is not mentioned in the pull request comment.

Reply:
This change is mentioned in the test cases and on the linked issue title.